docs/content/preview/manage/backup-restore/lightweight-db-clone.md

```diff
@@ -0,0 +1,258 @@
+    }
+]
+```
+
+You can find the `source_database_id` or `source_namespace_id` from the [YB-Master leader UI](../../../reference/configuration/default-ports/#servers) under the `/namespaces` end point.
+
+In this example, two clones were made of the source database `00004000000030008000000000000000` that are COMPLETE. The two clones are `testing_clone_db` and `dev_clone_db` and they each have a unique `seq_no` used to identify each clone operation from the same source database.
+
+You can check the status of a specific clone operation if you have both the `source_database_id` (YSQL) or `source_namespace_id`(YCQL) and the `seq_no` as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS list_clones 00004000000030008000000000000000 2
+```
+
+```output
+[
+    {
+        "aggregate_state": "COMPLETE",
+        "source_namespace_id": "00004000000030008000000000000000",
+        "seq_no": "2",
+        "target_namespace_name": "dev_clone_db",
+        "restore_time": "2024-08-09 21:42:55.048663"
+    }
+]
+```
+
+Use the `list_clones` command to check whether a clone operation completed successfully or not.
+
+Note that the cluster doesn't allow two clone operations to happen concurrently on the same source database.
+
+### Example
+
+{{% explore-setup-single-local %}}
+
+The following example uses [ysqlsh](../../../admin/ysqlsh/) to create a database clone to recover from an accidental table deletion.
+
+1. Start ysqlsh and create the database as follows:
+
+    ```sh
+    ./bin/ysqlsh
+    CREATE DATABASE production_db;
+    ```
+
+1. Create a snapshot schedule that produces a snapshot once a day (every 1,440 minutes), and retains it for three days (4,320 minutes) using the following command:
+
+    ```sh
+    ./bin/yb-admin --master_addresses ip1:7100,ip2:7100,ip3:7100 create_snapshot_schedule 1440 4320 ysql.production_db
+    ```
+
+1. Create a table `t1` and add some data as follows:
+
+    ```sql
+    \c production_db;
+    production_db=# CREATE TABLE t1 (k INT, v INT);
+    production_db=# INSERT INTO t1 (k,v) SELECT i,i%2 FROM generate_series(1,5) AS i;
+    production_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+    k | v
+   ---+---
+    1 | 1
+    2 | 0
+    3 | 1
+    4 | 0
+    5 | 1
+   (5 rows)
+    ```
+
+1. Determine the exact time when your database is in the correct state. This timestamp will be used to create a clone of the production database from the point when it was in the desired state. Execute the following SQL query to retrieve the current time in UNIX timestamp format:
+
+    ```sql
+    production_db=# SELECT (EXTRACT (EPOCH FROM CURRENT_TIMESTAMP)*1000000)::decimal(38,0);
+    ```
+
+    ```output
+         numeric
+     ------------------
+      1723243720285350
+      (1 row)
+    ```
+
+1. Drop the table `t1` simulating a user error dropping the table accidentally.
+
+    ```sql
+    production_db=# DROP TABLE t1;
+    ```
+
+    ```output
+    DROP TABLE
+    ```
+
```

Comment:
We can augment the example as follows:

Meanwhile, table t2 is still accepting reads/writes. We can simulate that by inserting 2 more rows as follows:
```sql
INSERT INTO t2 (key,c1) SELECT i,md5(random()::text) FROM generate_series(6,7) AS i;
```
After that, you try to query table `t1`. However, you figure out the table is dropped and there is no way you can query it.